cmake/find/FindSQLite3.cmake

```diff
@@ -0,0 +1,48 @@
+# ----------------------------------------------------------------------
+# FindSQLite3.cmake - SQLite3 CMake module
+#
+# Copyright (C) 2015 Alexandre Pretyman. All rights reserved.
+#
+# ----------------------------------------------------------------------
+
+if(HUNTER_STATUS_DEBUG)
+  message("[hunter] SQLITE3_ROOT ${SQLITE3_ROOT}") 
```

Comment:
Ok, will do, I was thinking it could be useful outside hunter, but maybe there is no point in that line of thought :-)